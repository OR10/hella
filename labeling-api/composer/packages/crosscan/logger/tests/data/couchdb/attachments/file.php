<?php
return array(
    'json' => array(
        'BarId' => array(
            'content_type' => 'image/png',
            'revpos' => 1,
            'length' => 285,
            'stub' => true,
        )
    ),
    'files' => array(
        'BarId' => file_get_contents( __DIR__ . '/../../../images/test.png' )
    )
);
