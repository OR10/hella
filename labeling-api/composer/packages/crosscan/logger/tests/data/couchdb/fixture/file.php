<?php
return array(
    'type' => 'File',
    'fixture' => new cscntLogFileStruct(
        'image/png',
        file_get_contents(
            __DIR__ . '/../../../images/test.png'
        )
    )
);