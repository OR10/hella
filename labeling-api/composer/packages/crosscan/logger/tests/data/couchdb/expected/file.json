{
    "mimeType": "image/png",
    "filesize": 285
}